### ISSUE TITLE
Doesn't detect Steam

### ISSUE THREAD

**Issue by sprite-1** [tkashkin/GameHub#199]
###### Expected behavior
It should detect Steam

###### Actual behavior
It doesn't detect it

###### Steps to reproduce
I installed Steam straight from the website via their DEB installer. This is on a fresh install of elementary OS 5.0. As you can see,Steam is indeed installed and running
![image_2019-02-14_21-57-52](https://user-images.githubusercontent.com/33685433/52791730-97bc9880-30a4-11e9-8b7d-9b8b08b0a351.png)
I even tried changing the Steam folder to point to ~/.local/share/Steam but it still can't detect it.

###### Version and environment

```
Version: 0.13.1-d48c261-dev
Branch:  dev
Commit:  d48c261 (d48c2616f49dfb6d52b291bcfd39e032dd0e686a)
Distro:  elementary OS 5.0 Juno
DE:      Pantheon
```

**Comment by tkashkin** [comment]
What is the output of `dpkg-query -W -f=\${Status} steam`?

**Comment by sprite-1** [comment]
> What is the output of `dpkg-query -W -f=\${Status} steam`?

`unknown ok not-installed`

**Comment by tkashkin** [comment]
If it's running on elementary, it expects `steam` package to be installed and opens AppCenter if package is not installed.

I will change this check now.

**Comment by tkashkin** [comment]
Should be fixed in 3254279.

**Comment by sprite-1** [comment]
Okay I just tried out the most latest release and now it's talking about 
![image](https://user-images.githubusercontent.com/33685433/52795158-27b21080-30ac-11e9-9d9a-2dbdd6c55458.png)
I've completely exited Steam and relaunched it but this message stays, should I open a new issue about this?
